### DISCUSSION TITLE
vmtool --action 支持的动作枚举文档没有说

### DISCUSSION THREAD

**Discussion by gentlesong** [alibaba/arthas#2139] [Q&A] [answered]
vmtool --action getInstances --className XXX 获得对象的实例
但是--action后面还有其他的动作支持吗？
测试了sumInstanceSize 和countInstances 都不行

**Comment by jdxia** [comment]
com.taobao.arthas.core.command.monitor200.VmToolCommand.VmToolAction
<img width="467" alt="image" src="https://user-images.githubusercontent.com/23069626/211797499-5effe638-4528-415b-b38f-db965a7a38d7.png">
源码里面有

**Comment by hengyunabc** [comment, marked as answer]
没有开放这两个功能，因为对jvm消耗太大。真想要调用，可以用 ognl 表达式来调。
